### ISSUE TITLE
Firewall ports need to be changed?

### ISSUE THREAD

**Issue by FrankdeRouw** [pascalopitz/unoffical-sonos-controller-for-linux#90]
Opened the ports as suggested in readme. When starting up it hangs on: "Searching for sonsos system". When I disable the firewall no problems when starting up. System: Linux mint 19.3

Naar                       Actie       Van
----                       -----       ---
139,445/tcp                ALLOW       Anywhere                  
137,138/udp                ALLOW       Anywhere                  
139,445/tcp (v6)           ALLOW       Anywhere (v6)             
137,138/udp (v6)           ALLOW       Anywhere (v6)             
3400 (v6)                  ALLOW       Anywhere (v6)             
1905 (v6)                  ALLOW       Anywhere (v6)             

139,445/tcp                ALLOW OUT   Anywhere                  
137,138/udp                ALLOW OUT   Anywhere                  
139,445/tcp (v6)           ALLOW OUT   Anywhere (v6)             
137,138/udp (v6)           ALLOW OUT   Anywhere (v6)             
1900 (v6)                  ALLOW OUT   Anywhere (v6)             
1400 (v6)                  ALLOW OUT   Anywhere (v6)

**Comment by pascalopitz** [comment]
Ah yes, that's be because node-sono's default ports are different.
Try:
- Incoming TCP: 4000
- Outgoing TCP: 1400
- Outgoing UDP: 1900

I think that should do. I'll update the readme.

**Comment by pascalopitz** [comment]
Actually, I have to correct myself here, having tested this out. The UDP discovery in node-sonos doesn't set a default incoming port and thus relies on a randomly assigned one. I have now set it to be 1905.

We end up with:

- Incoming TCP: 4000
- Outgoing TCP: 1400
- Incoming UDP: 1905
- Outgoing UDP: 1900

This required a code change, which I have decided marks the end of the beta period of v0.2.0. So, fixed as per v0.2.0.
